README.md

```diff
@@ -142,7 +150,7 @@ const AsyncHello = loadable(() => import('./Hello.js'))
 // Hello.js
 import { hot } from 'react-hot-loader'
 const Hello = () => 'Hello'
-export default hot(module)(MyComponent)
+export default hot(module)(MyComponent) // <-- the only change to do
```

Comment:
should `MyComponent` be `Hello` in this example?

Reply:
:P

Reply:
😆